app/src/main/java/org/jellyfin/androidtv/preference/UserSettingPreferences.kt

```diff
@@ -15,7 +15,7 @@ class UserSettingPreferences(
 ) {
 	companion object {
 		val skipBackLength = intPreference("skipBackLength", 10_000)
-		val skipForwardLength = intPreference("skipForwardLength", 30_000)
+		val skipForwardLength = intPreference("skipForwardLength", 10_000)
```

Comment:
The default value needs to be consistent with jellyfin-web

https://github.com/jellyfin/jellyfin-web/blob/9fbc37fbf30e986451964d3cd8fd178d1a3895be/src/scripts/settings/userSettings.js#L366

Reply:
i changed the value to 30 seconds